src/generateZeroFunction.js

```diff
@@ -12,7 +12,9 @@
  *
  */
 function generateZeroFunction() {
-  // write code here
+  return function second() {
```

Comment:
why function name is second? In this case you don't need a name, so maybe just remove it.